lib/Controller/RoomController.php

```diff
@@ -842,7 +842,10 @@ public function getParticipants(bool $includeStatus = false): DataResponse {
 		$results = [];
 
 		$statuses = [];
-		if ($this->userId !== null && $includeStatus && count($participants['users']) < 100) {
+		if ($this->userId !== null
+			&& $includeStatus
+			&& count($participants['users']) < 100
+			&& $this->appManager->isEnabledForUser('user_status')) {
```

Comment:
appmanager is already in the roomcontroller to check for circles, just in case someone is wondering